source/fundamentals/serialization/poco.txt

```diff
@@ -261,6 +261,167 @@ The following code sample maps the ``Identifier`` property to the
    The ``_id`` field mapping logic described in this section only applies to the
    root document and does not apply to nested documents.
 
+Specify an ID Generator
+``````````````````````
+
+Every document in a MongoDB collection must have a unique ID. When you serialize an object
+to a collection, if the ``Id`` property contains the default
+value for its data type (usually ``null``), the {+driver-short+} doesn't serialize the
+default value. Instead, the driver generates a unique ID value for the property.
+
+When you apply the ``[BsonId]`` attribute to a property, you can pass the ``IdGenerator``
+argument to specify the type of ID value the driver generates for the corresponding
```

Comment:
Not "the type of ID value"

The type is already determined by the type of the property.

What are you specifying is which generator to use to generate an Id value when needed.